### PR TITLE
Allow all cors

### PR DESCRIPTION
**Pull request by omnivion**
### Information
- JIRA story QPPSF-5378.

### Changes proposed in this PR:
- *(Please list what you changed.)*
-
-

### Checklist 
- [ ] All JUnit tests pass (`mvn clean verify`).
- [ ] New unit tests written to cover new functionality.
- [ ] Added and updated JavaDocs for non-test classes and methods.
- [ ] No local design debt. Do you feel that something is "ugly" after your changes?
- [ ] Updated documentation (`README.md`, etc.) depending if the changes require it.

### CODE REVIEW

**Comment by saquino0827** [comment]
@omnivion Once this is merged, we should update the web hosted s3 bucket to use dev or imp (probably imp because the vpn restrictions in dev). Also, we may want to consider merging `QPPSF-4883` into this one for the front end changes.

**Comment by saquino0827** [comment]
Follow up to please merge in the frontend changes from branch `QPPSF-4883` to this story.

File: frontend/src/app/app.component.ts [hunk 1]
```diff
@@ -2,7 +2,7 @@ import { Component, OnInit } from '@angular/core';
 import { FileSelectDirective, FileDropDirective, FileUploader } from '../../node_modules/ng2-file-upload/ng2-file-upload';
 import 'rxjs/Rx' ;
 
-const URL  = 'https://qpp.cms.gov/api/submissions/converter';
+const URL  = 'http://localhost:8080/';
```
Comment (omnivion) [inline frontend/src/app/app.component.ts]: is this accurate @saquino0827

Comment (saquino0827) [reply frontend/src/app/app.component.ts]: For Local host yes, I feel like this was pointed to dev-pre at some point. Let me check.

Comment (saquino0827) [reply frontend/src/app/app.component.ts]: The dev-pre URL is `https://preview.qpp.cms.gov/api/submissions/converter`. I believe if we wanted to test this in dev, we may have to get our s3 bucket configured to work through the VPN. And for IMP we would need to include the cookie. Localhost lets you deploy it locally and test.

**Pull request merged**